src/hotspot/share/logging/logAsyncWriter.hpp

```diff
@@ -135,13 +135,9 @@ class AsyncLogWriter : public NonJavaThread {
   class AsyncLogLocker;
 
   static AsyncLogWriter* _instance;
-  // _lock(1) denotes a critional region.
-  Semaphore _lock;
-  // _sem is a semaphore whose value denotes how many messages have been enqueued.
-  // It decreases in AsyncLogWriter::run()
-  Semaphore _sem;
   Semaphore _flush_sem;
-
+  os::PlatformMonitor _lock;
```

Comment:
Please add a comment before this:

// Can't use a Monitor here as we need a low-level API that can be used
// late in a thread's termination process.

Reply:
done.